Content.Server/Preferences/Managers/ServerPreferencesManager.cs

```diff
@@ -334,7 +334,7 @@ private async Task<PlayerPreferences> GetOrCreatePreferencesAsync(NetUserId user
             var prefs = await _db.GetPlayerPreferencesAsync(userId, cancel);
             if (prefs is null)
             {
-                return await _db.InitPrefsAsync(userId, HumanoidCharacterProfile.Random(), cancel);
+                return await _db.InitPrefsAsync(userId, HumanoidCharacterProfile.Random(new HashSet<string>(_cfg.GetCVar(CCVars.ICNewAccountSpeciesBlacklist).Split(","))), cancel);
```

Comment:
Too much is happening in a single line here to easily read, and it's also way past the line character limit. Either break this up into multiple lines, or perhaps better, take some of it out into a variable and read the variable